docs/source/actions.rst

```diff
@@ -61,15 +61,41 @@ Default: ``None``
 
 AWS Access Key.
 
+.. warning::
+
+    This setting has been deprecated in preference of ``SPIDERMON_AWS_ACCESS_KEY_ID``.
+
 .. _SPIDERMON_AWS_SECRET_KEY:
 
 SPIDERMON_AWS_SECRET_KEY
 ------------------------
 
+Default: ``None``
+
 AWS Secret Key.
 
+.. warning::
+
+    This setting has been deprecated in preference of ``SPIDERMON_AWS_SECRET_ACCESS_KEY``.
+
+.. SPIDERMON_AWS_ACCESS_KEY_ID:
+
+SPIDERMON_AWS_ACCESS_KEY_ID
+---------------------------
+
 Default: ``None``
 
+AWS Access Key. If not set, it defaults to `AWS_ACCESS_KEY_ID`_.
```

Comment:
It is not clear in this line that the default value is related to a Scrapy settings (without following the link). I believe that this could be clearer here.